diff --git a/ext/wasm/GNUmakefile b/ext/wasm/GNUmakefile
index a99513bfa..920e491ca 100644
--- a/ext/wasm/GNUmakefile
+++ b/ext/wasm/GNUmakefile
@@ -191,7 +191,8 @@ SQLITE_OPT = \
   '-DSQLITE_DEFAULT_UNIX_VFS="unix-none"' \
   -DSQLITE_USE_URI=1 \
   -DSQLITE_WASM_ENABLE_C_TESTS \
-  -DSQLITE_C=$(sqlite3.c)
+  -DSQLITE_C=$(sqlite3.c) \
+  "$(project_dir)/build/simple-master/build_static/src/libsimple.a"
 
 .NOTPARALLEL: $(sqlite3.h)
 $(sqlite3.h):
