diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 6d7474b..78a4d59 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -48,6 +48,7 @@
   target_include_directories(simple INTERFACE ${SQLITE3_HEADERS_DIR})
 endif()

+target_compile_options(simple PRIVATE -Oz -DSQLITE_CORE)
 if(BUILD_SQLITE3)
   target_link_libraries(simple PUBLIC coverage_config PRIVATE PINYIN_TEXT SQLite3)
 else()
